ci/plan.yml

```diff
@@ -111,6 +112,8 @@ VARIANTS:
       PLAIDML_EXPERIMENTAL: 1
       PLAIDML_DEVICE: llvm_cpu.0
       PLAIDML_TARGET: llvm_cpu
+      STRIPE_JIT: 1
```

Comment:
We only want to set `STRIPE_JIT` when targeting the `llvm_cpu.0` device

Reply:
I believe it is STRIPE_JIT=1 for llvm_cpu, not opencl_cpu